.gitlab-ci.yml

```diff
@@ -194,9 +219,9 @@ create-instances:
     - |
       if [ $INSTANCE_ROLE == "Server Master" ];
         then
-          ./Tests/scripts/slack_notifier.sh 'test_playbooks' $ARTIFACTS_FOLDER/env_results.json
+          echo "'test_playbooks' ${ENV_RESULTS_PATH} '${SLACK_CHANNEL}'" >> $ARTIFACTS_FOLDER/slack_notifier_input.txt
       fi
-    - python3 ./Tests/scripts/destroy_instances.py $ARTIFACTS_FOLDER $ARTIFACTS_FOLDER/env_results.json "$INSTANCE_ROLE" "$TIME_TO_LIVE"
+    - python3 ./Tests/scripts/destroy_instances.py $ARTIFACTS_FOLDER $ENV_RESULTS_PATH "$INSTANCE_ROLE" "$TIME_TO_LIVE"
```

Comment:
Depending on your decision with the slack notifier ⬆️ may need to change this bit.